full-moon/src/tokenizer/structs.rs

```diff
@@ -121,21 +120,28 @@ symbol! {
         Until => "until",
         While => "while",
 
-        [lua52 | luajit] Goto => "goto",
+        [lua52 | luajit | cfxlua] Goto => "goto",
+
+        [luau | cfxlua] PlusEqual => "+=",
+        [luau | cfxlua] MinusEqual => "-=",
+        [luau | cfxlua] StarEqual => "*=",
+        [luau | cfxlua] SlashEqual => "/=",
 
-        [luau] PlusEqual => "+=",
-        [luau] MinusEqual => "-=",
-        [luau] StarEqual => "*=",
-        [luau] SlashEqual => "/=",
         [luau] DoubleSlashEqual => "//=",
         [luau] PercentEqual => "%=",
-        [luau] CaretEqual => "^=",
+        [luau | cfxlua] CaretEqual => "^=",
         [luau] TwoDotsEqual => "..=",
 
         [luau | lua53] Ampersand => "&",
         [luau] ThinArrow => "->",
         [luau | lua52 | luajit] TwoColons => "::",
 
+        [cfxlua] LeftShift => "<<=",
+        [cfxlua] RightShift => ">>=",
+        [cfxlua] BitwiseAndAssignment => "&=",
+        [cfxlua] BitwiseOrAssignment => "|=",
+        [cfxlua] SafeNavigation => "?.",
```

Comment:
As mentioned above

```suggestion
        [cfxlua] DoubleLessThanEqual => "<<=",
        [cfxlua] DoubleGreaterThanEqual => ">>=",
        [cfxlua] AmpersandEqual => "&=",
        [cfxlua] PipeEqual => "|=",
        [cfxlua] QuestionMarkDot => "?.",
```

To align with the other options. We try to make the names represent the characters rather than the logic, since in future >>= could mean something other than right shift operator for a different Lua dialect